### PR TITLE
used black on all individual python stubs

### PR DESCRIPTION
**Pull request by jpwood**
Ran black on individual python sections for obvious reasons.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,16 +26,20 @@ For example, if your app was called `tester`, you could add the following to `te
...
+    facility="tester",  # name of program logging the message
+    graylog_address=("0.0.0.0", 9999),  # address for graylog (ip, port)
+    do_print=True,  # print each log to console
+    do_write=True,  # write each log to file
+    logfile="mylog.txt",  # custom path to logfile
+    line_length=200,  # line truncation for console logging
+    truncation=1000,  # longest possible value in extra data
+    private_data={"password"},  # set of sensitive args/kwargs
+    obscured="******",
+)  # string with which to obscure data
```
Comment (hukkinj1) [inline README.md]: this comment now on wrong line

File: README.md [hunk 2]
```diff
@@ -57,31 +61,32 @@ For an example use-case, let's make a simple class that multiplies two numbers,
 ```python
 @loggo
 class Multiplier(object):
```
Comment (hukkinj1) [inline README.md]: Not really in the scope, but if you wish, remove the `object` inheritance. We dont do python2...

File: README.md (hunk 1 above)
Comment (jpwood) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (jpwood) [reply README.md]: fixed

**Pull request merged**